metrics/cartridge/issues.lua

```diff
@@ -3,14 +3,18 @@ local fun = require('fun')
 
 local collectors_list = {}
 
-local function update_info_metrics()
+local function update()
     local list_on_instance = rawget(_G, '__cartridge_issues_list_on_instance')
 
     if not list_on_instance then
         return
     end
 
-    local issues = list_on_instance()
+    local ok, issues = pcall(list_on_instance)
+
+    if not ok then
+        return
```

Comment:
What about some log.error message?

Reply:
In case of stateboard it will log error each time metric will be collected, I don't think this is appropriate